src/collection/components/modals/CutFragmentModal.tsx

```diff
@@ -40,41 +42,90 @@ const CutFragmentModal: FunctionComponent<CutFragmentModalProps> = ({
 	const [playerTicket, setPlayerTicket] = useState<string>();
 	const [fragmentStartTime, setFragmentStartTime] = useState<number>(fragment.start_oc || 0);
 	const [fragmentEndTime, setFragmentEndTime] = useState<number>(
-		fragment.end_oc || toSeconds(itemMetaData.duration) || 0
+		fragment.end_oc || toSeconds(itemMetaData.duration, true) || 0
+	);
+	const [fragmentStartTimeString, setFragmentStartTimeString] = useState<string>(
+		formatDurationHoursMinutesSeconds(fragment.start_oc || 0)
+	);
+	const [fragmentEndTimeString, setFragmentEndTimeString] = useState<string>(
+		formatDurationHoursMinutesSeconds(
+			fragment.end_oc || toSeconds(itemMetaData.duration, true) || 0
+		)
 	);
 
+	const getValidationErrors = (): string[] => {
+		const start = toSeconds(fragmentStartTimeString, true);
+		const end = toSeconds(fragmentEndTimeString, true);
+
+		return getValidationErrorsForStartAndEndTime({
+			...fragment,
+			start_oc: start,
+			end_oc: end,
+		});
+	};
+
 	const onSaveCut = () => {
-		updateFragmentProperty(fragmentStartTime, 'start_oc', fragment.id);
-		updateFragmentProperty(fragmentEndTime, 'end_oc', fragment.id);
+		const errors = getValidationErrors();
+		if (errors && errors.length) {
+			toastService(
+				<>
+					{errors.map(error => (
+						<>
+							{error}
+							{errors.length > 1 ? (
+								<>
+									<br />
```

Comment:
Can't you use a spacer around error instead?

Reply:
Nee, heb ik geprobeerd, dan zet hij alle errors naast elkaar:
![image](https://user-images.githubusercontent.com/1710840/67679353-2bab6780-f989-11e9-9e37-60f52f1fcb3a.png)